jest.config.js

```diff
@@ -1,5 +1,23 @@
-module.exports = {
+const { pathsToModuleNameMapper } = require('ts-jest');
```

Comment:
Are we force to use this ? Feels kinda hacky

Reply:
But again it is fine. it is only the tests.

Reply:
Yeah this dev-dependencie is very usefull to make jest able to supports all features of TypeScript